clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -1,125 +1,361 @@
+    features: "Features",
+    features_description: "Below, you will find a list of the features",
+    feature_example: "Match and combine data",
+    feature_example_description: "Information about your activity",
+    special_features: "Special features",
+    special_feature_example: "Use precise geolocation data",
+    special_feature_example_description:
+      "With your acceptance, your precise location",
+    vendors: "Vendors",
+    vendors_description: "Below, you will find a list of vendors",
+    vendors_iab: "IAB TCF Vendors",
+    vendor_iab_example: "Captify",
+    vendor_iab_example_description:
+      "Captify stores cookies with a maximum duration",
+    vendors_other: "Other vendors",
+    vendor_other_example: "Fides System",
+    vendor_other_example_description:
+      "Fides System stores cookies with a maximum duration",
+    vendor_privacy_policy: "Privacy policy",
+    vendor_legint_disclosure: "Legitimate interest disclosure",
+    iab: "IAB TCF",
+    retention: "Retention",
+    consent: "Consent",
+    legint: "Legitimate interest",
+    data_categories: "Data categories",
+  },
+};
+
+/**
+ * Spanish translations for banner & modal
+ */
+const SPANISH_BANNER: TestBannerTranslations = {
+  banner_title: "[banner] Administrar sus preferencias de consentimiento",
+  banner_description: "[banner] Usamos cookies y métodos similares",
+  privacy_preferences_link_label: "Administrar preferencias",
+  reject_button_label: "No participar en nada",
+  accept_button_label: "Participar en todo",
+  acknowledge_button_label: "Aceptar",
+  gpc_label: "Control de privacidad global",
+  gpc_status_label: "Aplicado",
+  privacy_policy_link_label: "Política de privacidad",
+  privacy_policy_url: "https://privacy.example.com/",
+};
+
+const SPANISH_MODAL: TestModalTranslations = {
+  title: "Administrar sus preferencias de consentimiento",
+  description: "Usamos cookies y métodos similares",
+  save_button_label: "Guardar",
+  reject_button_label: "No participar en nada",
+  accept_button_label: "Participar en todo",
+  acknowledge_button_label: "Aceptar",
+  gpc_title: "Control de privacidad global detectado",
+  gpc_description:
+    "Su preferencia de control de privacidad global se ha respetado.",
+  gpc_label: "Control de privacidad global",
+  gpc_applied_label: "Aplicado",
+  gpc_overridden_label: "Anulado",
+  privacy_policy_link_label: "Política de privacidad",
+  privacy_policy_url: "https://privacy.example.com/",
+};
+
+const SPANISH_NOTICES: TestNoticeTranslations[] = [
+  {
+    title: "Mercadotecnia",
+    description: "Este sitio web usa cookies y servicios de mercadotecnia",
+  },
+  {
+    title: "Análisis",
+    description: "Este sitio web usa cookies analíticas y servicios",
+  },
+  {
+    title: "Esenciales",
+    description: "Este sitio web utiliza cookies esenciales y servicios",
+  },
+];
+
+/**
+ * Spanish translations for TCF banner & modal
+ */
+// TODO (PROD-1683): update i18n files
```

Comment:
There are a lot of `TODOs` in these strings, just like in the static `es/messages.json` file. Need to update with a new batch of translations